### DISCUSSION TITLE
One of THE BEST! Can it get better?

### DISCUSSION THREAD

**Discussion by DarksealStudios** [Gourieff/gourieff-roop#11] [Q&A]
I love how light and easy this is, thank you so much! I would just like to know if there's a way to bring some of those features from the automatic1111 extension to the portable version, like the male vs female detection. Love it! It's better than most.

**Comment by Gourieff** [comment]
Thanks for your good words!
Yeah, I have in plans to bring some feature form sd or comfy version to this app
I don’t know whether I’ll do it before NY or not, but it will definitely be implemented

**Comment by MonsieurPoole** [comment]
I love your app! It's phenomenal! Is there any way to get it to recognize when the target is eating a banana? It's the only thing I think it's missing.

**Comment by ZIX9900TOYOU** [comment]
run pythun is not working at all any idea how to fix it brother????
